### PR TITLE
Couple of changes to the document

### PR DESCRIPTION
**Pull request by Mohit-Nain**
Correcting a typo in line 53
Line 226 should be re-written as "To easily configure"

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/auth/generic-oauth.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ allowed_organizations =
 
 ## JMESPath examples
 
-To ease configuring a proper JMESPath expression you can test/evaluate expression with a custom payload at http://jmespath.org/.
+To easily configure a proper JMESPath expression you can test/evaluate expression with a custom payload at http://jmespath.org/.
```
Comment (aknuds1) [inline docs/sources/auth/generic-oauth.md]:
```suggestion
To ease configuration of a proper JMESPath expression, you can test/evaluate expressions with custom payloads at http://jmespath.org/.
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]
I made a suggestion that I think improves the language.

**Review by aknuds1 — APPROVED** [review]
LGTM

**Comment by oddlittlebird** [comment]
@aknuds1, can you figure out why the build is failing? Nothing in this PR should affect the bit that is failing.

**Comment by dprokop** [comment]
@oddlittlebird we see some failures(404) from Ubuntu packages registry. It's like that since early morning

**Comment by aknuds1** [comment]
@oddlittlebird Yep as @dprokop says, we are having unrelated build failures that I'm working to fix.

**Pull request merged**